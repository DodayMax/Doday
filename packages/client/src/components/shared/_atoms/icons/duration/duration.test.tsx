import * as React from 'react';
import { shallow } from 'enzyme';
import { Icons } from '@shared';

describe('Duration', () => {
  it('renders correctly without props', () => {
    const duration = shallow(<Icons.Duration />);

    expect(duration).toMatchSnapshot();
  });
});
