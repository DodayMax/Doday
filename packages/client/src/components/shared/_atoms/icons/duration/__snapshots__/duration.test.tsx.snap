// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Duration renders correctly without props 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <UNDEFINED />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <React.Fragment>
        <path
          d="M17.876 7.511v-4c0-.736-.597-1.333-1.334-1.333H7.653c-.736 0-1.333.597-1.333 1.333v4c0 .379.162.739.444.991l3.836 3.454-3.836 3.453a1.333 1.333 0 0 0-.444.991v4c0 .736.597 1.333 1.333 1.333h8.89c.736 0 1.333-.597 1.333-1.333v-4c-.001-.378-.163-.739-.445-.991l-3.835-3.453 3.835-3.454c.282-.252.444-.612.445-.99zM16.542 20.4H7.653v-.444h8.89v.444zm0-1.778H7.653V16.4l4.027-3.627c.27.094.562.094.831 0l4.031 3.627v2.222zm0-11.11l-4.026 3.626a1.262 1.262 0 0 0-.832 0L7.654 7.51v-4h8.888v4z"
        />
      </React.Fragment>,
      "data-doday-test": "DurationIcon",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <path
          d="M17.876 7.511v-4c0-.736-.597-1.333-1.334-1.333H7.653c-.736 0-1.333.597-1.333 1.333v4c0 .379.162.739.444.991l3.836 3.454-3.836 3.453a1.333 1.333 0 0 0-.444.991v4c0 .736.597 1.333 1.333 1.333h8.89c.736 0 1.333-.597 1.333-1.333v-4c-.001-.378-.163-.739-.445-.991l-3.835-3.453 3.835-3.454c.282-.252.444-.612.445-.99zM16.542 20.4H7.653v-.444h8.89v.444zm0-1.778H7.653V16.4l4.027-3.627c.27.094.562.094.831 0l4.031 3.627v2.222zm0-11.11l-4.026 3.626a1.262 1.262 0 0 0-.832 0L7.654 7.51v-4h8.888v4z"
        />,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "d": "M17.876 7.511v-4c0-.736-.597-1.333-1.334-1.333H7.653c-.736 0-1.333.597-1.333 1.333v4c0 .379.162.739.444.991l3.836 3.454-3.836 3.453a1.333 1.333 0 0 0-.444.991v4c0 .736.597 1.333 1.333 1.333h8.89c.736 0 1.333-.597 1.333-1.333v-4c-.001-.378-.163-.739-.445-.991l-3.835-3.453 3.835-3.454c.282-.252.444-.612.445-.99zM16.542 20.4H7.653v-.444h8.89v.444zm0-1.778H7.653V16.4l4.027-3.627c.27.094.562.094.831 0l4.031 3.627v2.222zm0-11.11l-4.026 3.626a1.262 1.262 0 0 0-.832 0L7.654 7.51v-4h8.888v4z",
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      "type": Symbol(react.fragment),
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <React.Fragment>
          <path
            d="M17.876 7.511v-4c0-.736-.597-1.333-1.334-1.333H7.653c-.736 0-1.333.597-1.333 1.333v4c0 .379.162.739.444.991l3.836 3.454-3.836 3.453a1.333 1.333 0 0 0-.444.991v4c0 .736.597 1.333 1.333 1.333h8.89c.736 0 1.333-.597 1.333-1.333v-4c-.001-.378-.163-.739-.445-.991l-3.835-3.453 3.835-3.454c.282-.252.444-.612.445-.99zM16.542 20.4H7.653v-.444h8.89v.444zm0-1.778H7.653V16.4l4.027-3.627c.27.094.562.094.831 0l4.031 3.627v2.222zm0-11.11l-4.026 3.626a1.262 1.262 0 0 0-.832 0L7.654 7.51v-4h8.888v4z"
          />
        </React.Fragment>,
        "data-doday-test": "DurationIcon",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <path
            d="M17.876 7.511v-4c0-.736-.597-1.333-1.334-1.333H7.653c-.736 0-1.333.597-1.333 1.333v4c0 .379.162.739.444.991l3.836 3.454-3.836 3.453a1.333 1.333 0 0 0-.444.991v4c0 .736.597 1.333 1.333 1.333h8.89c.736 0 1.333-.597 1.333-1.333v-4c-.001-.378-.163-.739-.445-.991l-3.835-3.453 3.835-3.454c.282-.252.444-.612.445-.99zM16.542 20.4H7.653v-.444h8.89v.444zm0-1.778H7.653V16.4l4.027-3.627c.27.094.562.094.831 0l4.031 3.627v2.222zm0-11.11l-4.026 3.626a1.262 1.262 0 0 0-.832 0L7.654 7.51v-4h8.888v4z"
          />,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M17.876 7.511v-4c0-.736-.597-1.333-1.334-1.333H7.653c-.736 0-1.333.597-1.333 1.333v4c0 .379.162.739.444.991l3.836 3.454-3.836 3.453a1.333 1.333 0 0 0-.444.991v4c0 .736.597 1.333 1.333 1.333h8.89c.736 0 1.333-.597 1.333-1.333v-4c-.001-.378-.163-.739-.445-.991l-3.835-3.453 3.835-3.454c.282-.252.444-.612.445-.99zM16.542 20.4H7.653v-.444h8.89v.444zm0-1.778H7.653V16.4l4.027-3.627c.27.094.562.094.831 0l4.031 3.627v2.222zm0-11.11l-4.026 3.626a1.262 1.262 0 0 0-.832 0L7.654 7.51v-4h8.888v4z",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        "type": Symbol(react.fragment),
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
